slmsuite/hardware/cameras/alliedvision.py

```diff
@@ -46,7 +51,7 @@ class AlliedVision(Camera):
 
     sdk = None
 
-    def __init__(self, serial="", verbose=True, **kwargs):
+    def __init__(self, serial="", pitch_um=None, verbose=True, **kwargs):
```

Comment:
Why is pitch_um an explicit argument in the sublcass when it’s not used in the subclass init?

Reply:
It is used in the superclass init. It's there to remind the user to fill it in as AlliedVision cameras don't return pixel size.